samples/wifi/throughput/sample.yaml

```diff
@@ -9,3 +9,43 @@ tests:
       - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf7002dk_nrf5340_cpuapp
     tags: ci_build
+  sample.nrf7002.iot_devices:
+    build_only: true
+    extra_args: OVERLAY_CONFIG=overlay-iot-devices.conf
+    integration_platforms:
+      - nrf7002dk_nrf5340_cpuapp
+    platform_allow: nrf7002dk_nrf5340_cpuapp
+    tags: ci_build
+    skip: true
```

Comment:
Can you please add a comment explaining why?

Reply:
We added `skip: true` to ensure that CI time is not impacted, allowing us to run Twister on these samples to measure memory footprint.

Reply:
I know :), I was referring to adding a comment in the YAML file above `skip: true`

Reply:
The explanation makes no sens to me. **Have you tried running twister on anything which has `skip: true`?** 
Such entry shouldn't be used at all in samples. It just messes with the CI an documentation. It tell twister to ALWAYS skip the build sample and makes it untestable in the CI. It is a way how upstream wanted to skip a broken builds to not break CI, but we have a quarantine for this now.

Reply:
As discussed https://github.com/nrfconnect/sdk-nrf/pull/14210#issuecomment-1970779858 we can:

* add an entry to sample.yaml without `skip: true`
* Add that case to quarantine-integration
* It will be part of nightly

Reply:
@PerMac Updated changes as per comments, please review.